Manito/Manito/Screens/Detail-Ing/DetailngCodebaseViewController.swift

```diff
@@ -0,0 +1,378 @@
+//  Manito
+//
+//  Created by creohwan on 2022/11/03.
+//
+
+import UIKit
+import SnapKit
+
+class DetailingCodebaseViewController: BaseViewController {
+
+    // MARK: - property
+    
+    private lazy var titleLabel: UILabel = {
+        var label = UILabel()
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 34)
+        return label
+    }()
+    private lazy var periodLabel: UILabel = {
+        var label = UILabel()
+        label.textColor = .grey002
+        label.font = .font(.regular, ofSize: 16)
+        return label
+    }()
+    private lazy var statusLabel: UILabel = {
+        var label = UILabel()
+        label.backgroundColor = .mainRed
+        label.layer.masksToBounds = true
+        label.layer.cornerRadius = 11
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 13)
+        label.textAlignment = .center
+        return label
+    }()
+    private var missionBackgroundView: UIView = {
+        var view = UIView()
+        view.backgroundColor = .darkGrey004
+        view.layer.borderWidth = 1
+        view.layer.cornerRadius = 10
+        return view
+    }()
+    private let missionTitleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "오늘의 개별 미션"
+        label.textColor = .darkGrey002
+        label.font = .font(.regular, ofSize: 14)
+        return label
+    }()
+    private lazy var missionContentsLabel: UILabel = {
+        var label = UILabel()
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 18)
+        return label
+    }()
+    private let informationTitleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "진행 정보"
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 16)
+        return label
+    }()
+    private let manitteeBackView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .darkGrey002
+        view.makeBorderLayer(color: .white)
+        return view
+    }()
+    private let manitteeImageView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .characterYellow
+        view.layer.cornerRadius = 49.5
+        return view
+    }()
+    private var manitteeIconView: UIImageView = {
+        var imageView = UIImageView()
+        imageView.image = ImageLiterals.icManiTti
+        return imageView
+    }()
+    private lazy var manitteeLabel: UILabel = {
+        var label = UILabel()
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 15)
+        return label
+    }()
+    private let listBackView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .darkGrey002
+        view.makeBorderLayer(color: .white)
+        return view
+    }()
+    private let listImageView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .characterPink
+        view.layer.cornerRadius = 49.5
+        return view
+    }()
```

Comment:
p3;

tapGesture 넣는 코드입니다 !

```suggestion
    private lazy var listImageView: UIView = {
        let view = UIView()
        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(pushFriendListViewController))
        view.backgroundColor = .characterPink
        view.layer.cornerRadius = 49.5
        view.addGestureRecognizer(tapGesture)
        return view
    }()
```

Reply:
> p3;
> 
> tapGesture 넣는 코드입니다 !

de87b8e 감사합니다~